manual/build_seafile/windows.md

```diff
@@ -6,28 +6,94 @@ The following compilation steps are only valid for versions 8.0 and above.
 
 Before compiling and developing, you need to install and configure the development environment
 
-1. Microsoft Visual Studio 2019
-
+1. Microsoft Visual Studio 2019 (Windows SDK 10.0.18362.0)
 2. Package manager tools: [vcpkg](https://docs.microsoft.com/en-us/cpp/build/vcpkg?view=vs-2019)
-
-3. Install wix[wix](https://github.com/wixtoolset/wix3/releases/tag/wix3111rtm)
-
-4. Install paraffin [paraffin](https://github.com/Wintellect/Paraffin/releases)
+3. Install wix[wix](https://github.com/wixtoolset/wix3/releases/tag/wix3111rtm) to c:/wix
+4. Install paraffin [paraffin](https://github.com/Wintellect/Paraffin/releases) and copy Paraffin.exe to c:/wix/bin
+5. Prepare signing certificate seafile.pfx to c:/certs
 
 ## Installing third-party libraries
 
 ```bash
-glib, curl, openssl, libevent, jansson, sqlite3, pthreads, getopt-win32
-Install curl：`vcpkg.exe install curl[core,openssl]:x64-windows`
+glib, curl, openssl(version1.1.1d), openssl-windows(version1.1.1d-1), libevent, jansson, sqlite3, pthreads, getopt-win32, quazip
+Install example：`vcpkg.exe install curl[core,openssl]:x64-windows`
+
 ```
 
 Install and configure Qt:
 
 1. Search and install Qt Visual Studio tools in the Visual Studio plugin repository
-2. Install Qt, choose qt-opensource-windows-x86-5.13.1.exe ([QT download link](http://download.qt.io/archive/qt/)).
+2. Install Qt5.15.1 ([QT download link](http://download.qt.io/archive/qt/)).
 3. Set Qt Visual Studio Tools. The setting is to tell Visual Studio 2019 Qt installation directory. Set Qt Version in Extension->Qt VS Tools-> Qt Options in Visual Studio 2019. Here you need to select the bin directory of Qt. The path of the parent directory. After configuring the Qt installation directory, you also need to set the Qt version used in the project properties of seafile-client. Select properties in the right-click menu of the project, and then select "Qt Project Settings" in the properties dialog box, and select the Version name set in Qt VS Tools just now for Qt Installation.
```

Comment:
这一大段需要优化一下。

3. Configure Qt Visual Studio Tools. This will tell VS about Qt installation path. In VS menu, open Extension -> Qt VS -> Qt Options. Here you can setup Qt versions and the corresponding installation paths. You should set the path to the parent folder of Qt "bin" folder.
4. Choose Qt version to be used in your project. In VS navigation pane, right click on seafile-client project and choose "Properties". Then click "Qt Project Settings" in the properties dialog. You should select the Qt version you just setup in the last step.